open_instruct/dpo_utils.py

```diff
@@ -170,6 +170,27 @@ def _get_batch_logps(
         return (per_token_logps * loss_mask).sum(-1)
 
 
+def process_batch(
```

Comment:
This confuses me a bit, but willing to trust. Not sure where its used.

Reply:
Ah, it's just like "input_ids_chosen"

Reply:
Yeah. This is written by claude btw XD